CMake/RandLAPACKConfig.cmake.in

```diff
@@ -7,16 +7,16 @@ set(RandLAPACK_VERSION_MINOR "@RandLAPACK_VERSION_MINOR@")
 set(RandLAPACK_VERSION_PATCH "@RandLAPACK_VERSION_PATCH@")
 set(RandLAPACK_VERSION_DEVEL "@RandLAPACK_VERSION_DEVEL@")
 
-# lapack++
-if (NOT lapackpp_DIR)
-    set(lapackpp_DIR @lapackpp_DIR@)
-endif()
-find_dependency(lapackpp)
-
 # randblas
 if (NOT RandBLAS_DIR)
     set(RandBLAS_DIR ${CMAKE_CURRENT_LIST_DIR})
 endif()
 find_dependency(RandBLAS)
 
+# lapack++
+if (NOT lapackpp_DIR)
+    set(lapackpp_DIR @lapackpp_DIR@)
+endif()
+find_dependency(lapackpp)
+
```

Comment:
@burlen, why does it help to have RandBLAS come before lapackpp in this context?

Reply:
because our RandBLAS cmake config provides blas++ install prefix and locates blas++ cmake config. using the information from the RandBLAS install means we don't have to provide it on the command line, i.e. -Dblaspp_DIR which simplifies and robustifies the build process.